### ISSUE TITLE
Review ingredient handling in settlements

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/SmartHarvestSE#317]
Nexus report

> Ingredients aren't being harvested in settlements, while foodstuffs are. This is with "settlements excepted" and "loot allowed items" active.
> Ingredient plants aren't being harvested, but food plants are.
> It's not just the one or two, it is all ingredient plants in settlement+ areas. If I whitelist the area, they get harvested.

**Comment by SteveTownsend** [comment]
I cannot reproduce this - in Whiterun, with settings described, both farm plants (yielding food) and flowers (yielding ingredients) get looted as expected.

Parking for now with request for more info from user.

**Comment by SteveTownsend** [comment]
Test evidence for fix in v3.10.0.0 - this reproduced with CACO in the Load Order.
```
2021-11-21 16:10:58.124    trace  33424 Process REFR 0x01000f09 with base object Blue Mountain Flower/0x000bcf3f
2021-11-21 16:10:58.124    trace  33424  object Blue Mountain Flower/0x000bcf3f has-ingredient=true
2021-11-21 16:10:58.124     info  33424 REFR 0x01000f09 Base Blue Mountain Flower/0x00077e1c Object Type ingredient
2021-11-21 16:10:58.124    trace  33424 check bound object Blue Mountain Flower/0x00077e1c with type 30 harvestable=true
2021-11-21 16:10:58.124    trace  33424 flora Blue Mountain Flower/0x000bcf3f has lootable Blue Mountain Flower/0x00077e1c of type 3
2021-11-21 16:10:58.124    trace  33424 TESFormHelper for Blue Mountain Flower/0x00077e1c set as ingredient
2021-11-21 16:10:58.124    trace  33424 Blue Mountain Flower/0x00077e1c has invalid scope 2
2021-11-21 16:10:58.125    trace  33424 TESFormHelper for Blue Mountain Flower/0x00077e1c set as ingredient
2021-11-21 16:10:58.125    trace  33424 TESValueForm for Blue Mountain Flower/0x00077e1c has 1
2021-11-21 16:10:58.125    trace  33424 check bound object Blue Mountain Flower/0x00077e1c with type 30 harvestable=true
2021-11-21 16:10:58.125    trace  33424 No V/W check for LootingType 1
2021-11-21 16:10:58.125    trace  33424 SmartHarvest Blue Mountain Flower/0x000bcf3f for REFR 0x01000f09, collectible=false
```
